nilearn/plotting/img_plotting.py

```diff
@@ -738,3 +738,86 @@ def display_factory(display_mode):
                                 **kwargs)
 
     return display
+
+
+def plot_connectome(adjacency_matrix, node_coords,
+                    node_color='auto', node_size=50,
+                    edge_cmap=cm.bwr, edge_threshold=None,
+                    edge_kwargs=None, node_kwargs=None,
+                    output_file=None, display_mode='ortho',
+                    figure=None, axes=None, title=None,
+                    annotate=True, black_bg=False,
+                    alpha=0.7, **kwargs):
```

Comment:
I would avoid having "**kwargs" and use a "glass_brain_kwargs" rather, as it is not possible to guess from reading the function signature and docstring what these do.

Also, do you really need these kwargs? I have the feeling that the usecases for them will be minor, and that advanced users that want to tweak everything should rather instanciate a glass brain and use the methods of the corresponding object.